### PR TITLE
Fix plugin info in MANIFEST.MF

### PR DESCRIPTION
**Pull request by chanseokoh**

### CODE REVIEW

**Comment by chanseokoh** [comment]
`BasePluginXmlTest` has evolved to include tests for `build.properties`, so I think it is not terrible wrong to add tests for `MANIFEST.MF`. Moreover, in our setting, we put localized messages of `MANIFEST.MF` into `plugin.properties`, which `BasePluginXmlTest` loads and tests, so it is easy to do so there.

However, we also have an annoyance due to `BasePluginXmlTest` becoming a uber test class: #1281

**Comment by chanseokoh** [comment]
For now, I can add a test to `BasePluginXmlTest`. We can think about #1281 later.

**Comment by chanseokoh** [comment]
Add a MANIFEST.MF localization test. Let's wait and see if any bundle fails.

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java [hunk 1]
```diff
@@ -156,4 +163,32 @@ private void assertPropertyDefined(String name) {
...
+    boolean localizedMessageExists = false;
+    boolean bundleLocalizationDefined = false;
+
+    Attributes attributes = getManifestAttributes();
+    for (Entry<?, ?> entry : attributes.entrySet()) {
+      String key = entry.getKey().toString();
+      String value = entry.getValue().toString();
+      assertPropertyDefined(value);
+
+      localizedMessageExists |= value.startsWith("%");
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java]: This is hard to follow. Is there a more obvious, less clever way to write this test?

Comment (chanseokoh) [reply plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java]: I'll unroll this into a regular `if` statement.

**Comment by chanseokoh** [comment]
PTAL

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java [hunk 2]
```diff
@@ -156,4 +163,36 @@ private void assertPropertyDefined(String name) {
...
+        bundleLocalization = value;
+      }
+    }
+
+    if (localizedMessageExists) {
+      assertEquals("plugin", bundleLocalization);
+    }
+  }
+
+  private Attributes getManifestAttributes() throws IOException {
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java]: static?

File: plugins/com.google.cloud.tools.eclipse.appengine.standard.java8.test/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -4,7 +4,7 @@ Bundle-Name: Google Cloud Tools for App Engine Standard Java 8 Tests
 Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.standard.java8.test
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.standard.java8
 Bundle-Version: 0.1.0.qualifier
-Bundle-Vendor: Google, Inc.
+Bundle-Vendor: Google Inc.
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.standard.java8.test/META-INF/MANIFEST.MF]: If we do care about this, we should probably add a test for it to the BasePluginXmlTest.

File: plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java (hunk 2 above)
Comment (chanseokoh) [reply plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java]: Done.

File: plugins/com.google.cloud.tools.eclipse.appengine.standard.java8.test/META-INF/MANIFEST.MF (hunk 3 above)
Comment (chanseokoh) [reply plugins/com.google.cloud.tools.eclipse.appengine.standard.java8.test/META-INF/MANIFEST.MF]: Added a test. Let's see if all the tests will succeed.
